techniques/system/rudderApache/1.0/apache/main.cf

```diff
@@ -29,13 +29,15 @@ bundle common rudder_apache {
 
 bundle agent system_rudder_apache_configuration {
   methods:
-      "any" usebundle => enable_reporting;
```

Comment:
why do you remove that ?

Reply:
Not needed since we have the option to enable the reporting in the metadata <USEMETHODREPORTING>true</USEMETHODREPORTING>